src/acoupi/system/programs.py

```diff
@@ -168,6 +174,59 @@ def write_program_file(
     )
 
 
+def get_program_store(settings: Settings) -> types.Store:
+    """Get the store for the program."""
+    db_path = Path(
+        json.load(settings.program_config_file.open())["paths"]["db_metadata"]
+    )
+    store = SqliteStore(db_path=db_path)
+    return store
+
+
+def end_program(settings: Settings) -> None:
+    """End the current program.
+
+    After stopping and disabling the systemd services, the deployment is ended.
+    To ensure the correct restarting of the next program, the end_program() function
+    ensure that remaining tasks are completed when the program is stopped.
+
+    The tasks to check are:
+    - file_management_task (if implemented). Check if there are remaning files in the
+    temprary directory and move them to the correct directory.
+    - detection_task (if implemented). Check if there are remaning files to be processed
+    and process them.
+    """
+    tmp_audio_path = Path(
+        json.load(settings.program_config_file.open())["paths"]["tmp_audio"]
+    )
+    recordings = list(tmp_audio_path.glob("*"))
+
+    if len(recordings) > 0:
+        print(
```

Comment:
@mbsantiago Unsure about this print statement! It's handy as it will print it out in the terminal and so, a user knows why it takes a little time for the system to shut down. But wonder, if there would be a better option.

Reply:
As it is in the BasicProgram(). The print is now capture by the logger.info() .